[2025-05-22T07:39:18.089+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: log_etl_pipeline.load_to_db manual__2025-05-22T07:39:15.337713+00:00 [queued]>
[2025-05-22T07:39:18.099+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: log_etl_pipeline.load_to_db manual__2025-05-22T07:39:15.337713+00:00 [queued]>
[2025-05-22T07:39:18.100+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-22T07:39:18.114+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_to_db> on 2025-05-22 07:39:15.337713+00:00
[2025-05-22T07:39:18.119+0000] {standard_task_runner.py:60} INFO - Started process 184 to run task
[2025-05-22T07:39:18.123+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'log_etl_pipeline', 'load_to_db', 'manual__2025-05-22T07:39:15.337713+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/log_etl_dag.py', '--cfg-path', '/tmp/tmp_d4m4ra5']
[2025-05-22T07:39:18.126+0000] {standard_task_runner.py:88} INFO - Job 12: Subtask load_to_db
[2025-05-22T07:39:18.191+0000] {task_command.py:423} INFO - Running <TaskInstance: log_etl_pipeline.load_to_db manual__2025-05-22T07:39:15.337713+00:00 [running]> on host d8cd3fbf198d
[2025-05-22T07:39:18.282+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='log_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_to_db' AIRFLOW_CTX_EXECUTION_DATE='2025-05-22T07:39:15.337713+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-22T07:39:15.337713+00:00'
[2025-05-22T07:39:18.283+0000] {logging_mixin.py:188} INFO - Loading data into PostgreSQL...
[2025-05-22T07:39:18.284+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-22T07:39:18.295+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=log_etl_pipeline, task_id=load_to_db, execution_date=20250522T073915, start_date=20250522T073918, end_date=20250522T073918
[2025-05-22T07:39:18.343+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-22T07:39:18.371+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
